src/components/views/elements/AppTile.tsx

```diff
@@ -596,9 +596,10 @@ export default class AppTile extends React.Component<IProps, IState> {
             "microphone; camera; encrypted-media; autoplay; display-capture; clipboard-write; " + "clipboard-read;";
 
         const appTileBodyClass = classNames({
-            mx_AppTileBody: !this.props.miniMode,
-            mx_AppTileBody_mini: this.props.miniMode,
-            mx_AppTile_loading: this.state.loading,
+            "mx_AppTileBody": true,
+            "mx_AppTileBody--normal": !this.props.miniMode,
```

Comment:
"Normal" is a somewhat ambiguous term. How about `--large`?

Reply:
It is surely better than `--normal`, though I now think it probably rather should be `mx_AppTileBody--notMini` if we are going to be super strict :-D

Reply:
Addressed with https://github.com/matrix-org/matrix-react-sdk/pull/11002/commits/9a5b3531c59e4f70ef57856c168557663812c161